Moved bars to a fragment

BadLog1n opened the pull request.

BadLog1n committed:
Moved bars to a fragment

MeeWeeL commented on app/src/main/res/layout/new_fragment_main.xml:
```diff
@@ -5,11 +5,61 @@
         android:id="@+id/mainFragmentRecyclerView"
         android:layout_width="match_parent"
-        android:layout_height="match_parent"
+        android:layout_height="wrap_content"
```
Нет. Высоту регулировать надо через ConstraintLayout, а значение указывается 0dp. При пустом списке, размер ресайклера будет 0, а значит не будет работать RefreshLayout. и наоборот, при переполненном списке как именно будет вести себя ресайклер?

MeeWeeL commented on app/src/main/res/layout/new_fragment_main.xml:
```diff
@@ -5,11 +5,61 @@
+        </com.google.android.material.appbar.CollapsingToolbarLayout>
+    </com.google.android.material.appbar.AppBarLayout>
+
+    <FrameLayout
```
ЛоадингЛэйаут. Его смысл - Перекрывать экран и предотвращать нажатие на остальное во время загрузки. В разметке порядок элементов определяет, кто на кого ложится поверх. В данном случае ЛоадингЛэйаут будет под ресайклером и его даже видно не будет.

MeeWeeL commented on app/src/main/res/layout/new_fragment_main.xml:
```diff
@@ -5,11 +5,61 @@
+        app:labelVisibilityMode="labeled"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:menu="@menu/navigation_bar_menu" />
+
```
Перед закрывающим Тэгом в разметки не нужна пустая строка. Можешь удалить в разметке вообще все пустые строки и прожать Ctrl+Alt+L и посмотреть, как расставится.

BadLog1n committed:
FIxed bars in main fragment

1. Height adjustment via ConstraintLayout
2. Moving loadingLayout down
3. Removed empty lines

BadLog1n committed:
Merge branch 'dev' into 1007_moveBarsFrag_to_dev
